module-example/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "powerhouse-switchboard-plugin-demo",
+    "version": "0.0.1",
+    "description": "Open-source API over database models made to be developer- and analyst-friendly.",
+    "author": "makerdao",
+    "type": "module",
+    "license": "AGPL-3",
+    "scripts": {
+        "dev": "vite-node -w src/index.ts",
+        "start": "NODE_ENV=production vite-node src/index.ts",
+        "debug": "DEBUG=1 npm run dev",
+        "lint": "eslint \"./**/*.{ts,tsx}\" --max-warnings=0",
+        "typecheck": "tsc --noEmit"
+    },
+    "dependencies": {
+        "@acaldas/document-model-libs": "^1.0.15",
+        "@apollo/server": "^4.5.0",
+        "@prisma/client": "^4.11.0",
+        "@types/cookie-parser": "^1.4.3",
+        "body-parser": "^1.20.2",
+        "cookie-parser": "^1.4.6",
+        "cors": "^2.8.5",
+        "dotenv": "^16.0.3",
+        "ethers": "^5.7.2",
+        "express": "^4.18.2",
+        "graphql": "^16.7.1",
+        "graphql-middleware": "^6.1.33",
+        "graphql-playground-middleware-express": "^1.7.23",
+        "jsonwebtoken": "^9.0.0",
+        "ms": "^2.1.3",
+        "nexus": "^1.3.0",
+        "nexus-validation-plugin": "^0.2.0",
+        "pino": "^8.11.0",
+        "pino-http": "^8.3.3",
+        "pino-pretty": "^10.0.0",
+        "siwe": "^2.1.4",
+        "vite-node": "^0.29.2",
+        "wildcard-match": "^5.1.2",
+        "zod": "^3.21.4"
+    },
+    "devDependencies": {
```

Comment:
Do you actually need all of those dependencies and devDependencies? Otherwise, please clean them up

Reply:
cleaned up according to `depcheck`